src/other/mlpmixer.jl

```diff
@@ -0,0 +1,69 @@
+# Utility function for creating a residual block with LayerNorm before the residual connection
+_residualprenorm(planes, fn) = SkipConnection(Chain(fn, LayerNorm(planes)), +)
+
+# Utility function for 1D convolution
+_conv1d(inplanes, outplanes, activation) = Conv((1, ), inplanes => outplanes, activation)
+
+"""
+    feedforward(planes, expansion_factor = 4, dropout = 0., dense = Dense)
+
+Feedforward block in the MLPMixer architecture.
+([reference](https://arxiv.org/pdf/2105.01601)).
+
+# Arguments
+  `planes`: Number of dimensions in the input and output.
+  `expansion_factor`: Determines the number of dimensions in the intermediate layer.
+  `activation`: Activation function to use.
+  `dropout`: Dropout rate.
+  `dense`: Type of dense layer to use in the feedforward block.
+"""
+function feedforward(planes, expansion_factor = 4, dropout = 0., dense = Dense)
```

Comment:
`mlpblock` instead? Is this different from ViT? If not, then it can go in `layers.jl` file too. Let's make the activation customizable then?

Reply:
I think the more we can reduce code duplication, the better. I would adopt the ViT interface for this function, then just pass in `expansion_factor * planes` in the MLPMixer builder code.

Reply:
That should be fine. I'll do that